pytest_rts/utils/common.py

```diff
@@ -190,7 +190,8 @@ def save_mapping_data(test_function_id, cov_data, testfiles, db_helper):
     for filename in cov_data.measured_files():
         src_file = os.path.relpath(filename, os.getcwd())
         conditions = [
-            "pytest-rts" in filename,
+            "init_phase_plugin.py" in filename,
+            "update_phase_plugin.py" in filename,
```

Comment:
The checks are fragile. Please be more specific, not just look for substring. Also: put excluded file names in some constant, defined at the top.

Reply:
Now that I think of it, would going back to the [.coveragerc file](https://coverage.readthedocs.io/en/coverage-5.3/source.html) make sense? With that, the user could define which paths to omit instead of me trying to guess what should be considered a source file. It sort of works automatically if you just have it in the root of the target repo.